apps/website/src/pages/docs/progressindicator.mdx

```diff
@@ -36,3 +36,9 @@ import * as AllExamples from 'examples';
 ### Props
 
 <PropsTable path={frontmatter.propsPath2} />
+
+## Overlay
+
+<LiveExample src='Overlay.main.tsx' truncate={false}>
```

Comment:
Add a little bit of text explaining that users can use progress indicators with overlay.

Reply:
Or remove at all as you added new mdx file.

Reply:
done :)